uint8 CO2_PROCESSING_IDLE = 0
uint8 CO2_PROCESSING_AIR_COLLECTION = 1
uint8 CO2_PROCESSING_MOISTURE_REMOVAL = 2
uint8 CO2_PROCESSING_ADSORPTION_BED1 = 3
uint8 CO2_PROCESSING_HUMIDIFICATION = 4
uint8 CO2_PROCESSING_TRANSFER_BED2 = 5
uint8 CO2_PROCESSING_DESORPTION = 6
uint8 CO2_PROCESSING_TO_SPACE = 7
uint8 CO2_PROCESSING_TO_SABATIER = 8



uint8 SYSTEM_HEALTH_UNKNOWN = 0
uint8 SYSTEM_HEALTH_GOOD = 1
uint8 SYSTEM_HEALTH_FAILURE = 2



uint8 co2_processing_state 
uint8 system_health
string data